config/application.rb

```diff
@@ -216,8 +220,22 @@ class Application < Rails::Application
     config.session_store :active_record_store, :secure => !!SETTINGS[:require_ssl]
   end
 
-  def self.setup_console
-    ENV['IRBRC'] = File.expand_path('../irbrc', __FILE__)
-    puts "For some operations a user must be set, try User.current = User.first"
+  class << self
+    def setup_console
+      ENV['IRBRC'] = File.expand_path('../irbrc', __FILE__)
+      puts "For some operations a user must be set, try User.current = User.first"
+    end
+
+    def dynflow
+      return @dynflow if @dynflow.present?
```

Comment:
when will that be ever set? and if it is already set, must we require and init it? i'm worried on a secnario when its defined, but did not call require or initialize.